css/main.css

```diff
@@ -420,6 +432,105 @@ header a {
 }
 
 @media (max-width: 767px) {
+  .project-pop-up-window {
+    background: #c1c7d0;
+    mix-blend-mode: multiply;
+    position: fixed;
+    top: 0;
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+  }
+
+  .project-pop-up-window > div > img {
+    display: none;
+  }
+
+  .project-pop-up-window > div {
+    padding: 16px;
+    background-color: white;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    width: 80vw;
+    height: 95vh;
+    overflow: auto;
+  }
+
+  .project-pop-up-window .close-item {
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    min-height: 25vh;
+    position: relative;
+  }
+
+  .project-pop-up-window .close-item img {
+    position: absolute;
+    top: 5%;
+    right: 5%;
+  }
+
+  .project-pop-up-window h2 {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .project-pop-up-window p {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .project-pop-up-window ul {
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+  }
```

Comment:
Great job @mohamed20163858 🤝

- [ ] You may also change the color of the closing icon to your mobile popup. As required in the project requirements we are supposed to close using this button to return to the main page but unfortunately, it is not visible enough. You could change it to a darker color.
![cancel button](https://user-images.githubusercontent.com/63917725/180093271-80a72928-a860-4697-9e8c-337d9065b155.PNG)

Reply:
@1GORDON actually there was a different background image with a different close button which was white but the problem here is in my javascript file there was no event listener for resizing popup window manually as it was totally static i solved it by adding an event listener to  detect any resize change and change the styling depend on the width dimension belong to mobile or desktop thank again for mention that